docs/sites.yml

```diff
@@ -1522,3 +1522,207 @@
+  main_url: https://www.creditloan.com/bad-credit-loans/
+  categories: 
+    - Loans
+    - Credits
+  featured: false
+- title: Financial Center
+  description: > 
+     Member-owned, not-for-profit, co-operative whose members receive financial benefits in the form of lower loan rates, higher savings rates, and lower fees than banks.
+  main_url: https://fcfcu.com/
+  categories: 
+    - Loans
+    - Credits
+    - Online Banking
+    - Nonprofit
+  featured: false
+- title: Open FDA
+  description: > 
+     Provides APIs and raw download access to a number of high-value, high priority and scalable structured datasets, including adverse events, drug product labeling, and recall enforcement reports.
+  main_url: https://fcfcu.com/
+  categories: 
+    - APIs
+    - Datasets
+    - Reports
+  featured: false
+- title: Office of Institutional Research and Assessment
+  description: > 
+     Good Data, Good Decisions
+  main_url: http://oira.ua.edu/
+  categories: 
+    - Research
+    - Data
+  featured: false
+- title: GM Capital One
+  description: > 
+     Introducing the new online experience for your GM Rewards Credit Card
+  main_url: https://gm.capitalone.com/
+  categories: 
+    - Rewards
+    - Credit Card
+  featured: false
+- title: House Manager
+  description: > 
+     Home service membership that offers proactive and on-demand maintenance for homeowners
+  main_url: https://housemanager.calstate.aaa.com/
+  categories: 
+    - Home
+    - Services
+  featured: false
+- title: Trintellix
+  description: > 
+     It may help make a difference for your depression (MDD).
+  main_url: https://us.trintellix.com/
+  categories: 
+    - Health
+    - Help
+  featured: false
+- title: The Telegraph Premium
+  description: > 
+     Exclusive stories from  award-winning journalists
+  main_url: https://premium.telegraph.co.uk/
+  categories: 
+    - Landing
+    - Newspaper
+    - Subscription
+  featured: false
+- title: html2canvas
+  description: > 
+     Screenshots with JavaScript
+  main_url: http://html2canvas.hertzen.com/
+  source_url: https://github.com/niklasvh/html2canvas/tree/master/www
+  categories: 
+    - Tool
+    - Screenshots
+    - JavaScript
+    - Documentation
+  built_by: Niklas von Hertzen
+  built_by_url: http://hertzen.com/
+  featured: false
+- title: The State of JavaScript 2017
+  description: > 
+      Data from over 20,000 developers, asking them questions on topics ranging from front-end frameworks and state management, to build tools and testing libraries.
+  main_url: https://stateofjs.com/
+  source_url: https://github.com/StateOfJS/StateOfJS
+  categories: 
+    - Data
+    - JavaScript
+    - Statistics
+  built_by: StateOfJS
+  built_by_url: https://github.com/StateOfJS/StateOfJS/graphs/contributors
+  featured: false
+- title: Dato CMS
+  description: > 
+      The API-based CMS your editors will love
+  main_url: https://www.datocms.com/
+  source_url: https://github.com/StateOfJS/StateOfJS
```

Comment:
Looks like this source_url is duplicated a few times?

Reply:
my bad on copy-pasting, in fact since the beggining of the _List_ I was thinking on a _Safe_ way to include new websites without repetition and with a way to guarantee kinds of _categories_. Still thinking on it.